src/styles.css

```diff
@@ -0,0 +1,22 @@
+.form-container {
+  width: 30rem;
+  padding: 2rem;
+  position: absolute;
+  top: 10rem;
+  left: 20rem;
+  border: 0.1rem solid;
+}
+
+.btn-div {
```

Comment:
Both `margin-top` and `text-align` are a part of Bootstrap's utility classes. Use those instead.